head.html

```diff
@@ -2,3 +2,4 @@
 <script src="/cigaradvisor/scripts/aem.js" type="module"></script>
 <script src="/cigaradvisor/scripts/scripts.js" type="module"></script>
 <link rel="stylesheet" href="/cigaradvisor/styles/styles.css"/>
+<link crossorigin="anonymous" rel="stylesheet" id="ca-alfaslabmonterrat-font-css" href="//fonts.googleapis.com/css?family=Alfa+Slab+One%7CMontserrat%3A300%2C400%2C500%2C600%2C700%2C800%2C900%7COpen+Sans%3A400%2C600%2C700&#038;ver=5.5.1" type="text/css" media="all" />
```

Comment:
this doesn't follow the best practices for loading fonts. See the docs on font fallback and and the use of the fonts.css

We can add this to the global styles.

Reply:
This is fixed.

Reply:
Just stumbled upon this after i did this myself. Provided my integration off all the fonts in #47. Might make sense to sync on this. With my changes you should be able to just use `font-family: var(--ff-alfaslabone)`